doc/examples/viz_widgets.py

```diff
@@ -121,9 +121,9 @@ def text_clicked(obj, event):
 Uncomment the following lines to start the interaction.
 """
 
-# show_manager.initialize()
-# show_manager.render()
-# show_manager.start()
+show_manager.initialize()
```

Comment:
Please don't reply to these comments, unless there is something to discuss. I get an email every time you make a comment like this one.